cfy_cluster_setup/main.py

```diff
@@ -50,6 +57,18 @@ def __init__(self,
         self.key_path = join(CERTS_DIR, self.name + '_key.pem')
 
 
+def _exception_handler(type_, value, traceback):
+    error = type_.__name__
+    if str(value):
+        error = '{0}: {1}'.format(error, str(value))
```

Comment:
```suggestion
        error = '{0}: {1}'.format(error, value)
```
str.format already does `str`

Reply:
What does this do that logger.exception doesn't already do?

Reply:
Well, `logger.error(err)` will just log the error without the error name. `logger.excpetion(err)` will also print out the traceback.
Here, we also print the error name in `logger.error`, and the traceback only if the level of the logger is `DEBUG`.

`In [16]: try: 
    ...:     1/0
    ...: except ZeroDivisionError as err:
    ...:     logging.error(err)
    ...:              
ERROR:root:integer division or modulo by zero

In [17]: try: 
    ...:     1/0
    ...: except ZeroDivisionError as err:
    ...:     logging.exception(err)
    ...:                  
ERROR:root:integer division or modulo by zero
Traceback (most recent call last):
  File "<ipython-input-17-f21fe5405855>", line 2, in <module>
    1/0
ZeroDivisionError: integer division or modulo by zero`

Reply:
I'm wondering if that's worth the extra code, or if we should maybe let the traceback be logged when such an error occurs?